client/src/pages/Bookings/Booking/Booking.tsx

```diff
@@ -0,0 +1,81 @@
+import { Grid, Stack, Typography, Paper } from '@mui/material';
+import AvatarDisplay from '../../../components/AvatarDisplay/AvatarDisplay';
+import { User } from '../../../interface/User';
+import EditBooking from './EditBooking';
+import { RequestStatus } from '../../../types/RequestStatus';
+
+interface Props {
+  start: string;
+  end: string;
+  userId: User;
```

Comment:
```suggestion
  user: User;
```
This seems like a user object and not just the id - ditto throughout :)